.ci.yaml

```diff
@@ -130,6 +130,10 @@ targets:
           "builder",
           "git"
         ]
+      ignore_cache_paths: >-
```

Comment:
Reminder to update https://github.com/flutter/cocoon/blob/main/CI_YAML.md#properties. It looks like some of the other properties might also be missing docs there?

Reply:
You got it! I will make sure to open that before submitting.